OpencartTest/src/test/UnsuccessfulReviewTest.java

```diff
@@ -76,12 +81,35 @@ public void unsuccessfullyWritingReviewBecauseOfIncorrectTextAndName(String inco
     @Test(priority = 4)
     public void unsuccessfullyWritingReviewBecauseOfEmptyNameAndTextFields(String emptyNameField, String emptyTextField, String messageOfUndeliveredReviewBecauseOfIncorrectText) {
         JavascriptExecutor js = (JavascriptExecutor) driver;
-        WebElement product = driver.findElement(By.cssSelector("img[alt='MacBook']"));
-        js.executeScript("arguments[0].scrollIntoView();", product);
-        product.click();
+        js.executeScript("arguments[0].scrollIntoView();", testProduct);
+        testProduct.click();
```

Comment:
why not make a separate method that will combine js.executeScript and click ? 
otherwise there's a lot of duplicate code